helmfile.d/config/mongo-express-common.yaml.gotmpl

```diff
@@ -0,0 +1,18 @@
+environment: {{ .Environment.Name }}
+
+image:
+  tag: v0.1.0
+
+basicauth:
+{{ if eq .Environment.Name "production" }}
```

Comment:
can we remove this "if" by having a default/common false, and a prod (in the prod file) true ?

Reply:
good point